maven_install_2.12.json

```diff
@@ -62,15 +62,15 @@
             {
                 "coord": "co.fs2:fs2-core_2.12:2.5.6",
                 "dependencies": [
-                    "org.scala-lang:scala-library:2.12.14",
                     "org.typelevel:cats-effect_2.12:2.5.1",
+                    "org.scala-lang:scala-library:2.12.15",
```

Comment:
is this needed?

Reply:
this is a generated file and we need to regenerate it. I’m not quite sure what determines these versions but we cannot change them beyond regenerating.